apps/emqx/src/emqx_channel.erl

```diff
@@ -2029,7 +2030,8 @@ enrich_connack_caps(
         'Topic-Alias-Maximum' => MaxAlias,
         'Wildcard-Subscription-Available' => flag(Wildcard),
         'Subscription-Identifier-Available' => 1,
-        'Shared-Subscription-Available' => flag(Shared)
+        'Shared-Subscription-Available' => flag(Shared),
+        'Receive-Maximum' => ReceiveMaximum
```

Comment:
nit, maybe do not set it if it is `65535` (upper limit) as it makes the `connack` a little bit larger.

Reply:
we have a default value (32)